### ISSUE TITLE
Question > Release versions

### ISSUE THREAD

**Issue by Musikolo** [cshamrick/stsauth#3]
Hi,

This is not an issue, just a question. I don't know about any better place where to ask it. 

Do you have any plan to release versions? I think that would be very nice and help better track when a version is done version in progress. Thoughts?

Thank you!

**Comment by cshamrick** [comment]
I have been deploying the packaged releases to pypi [here](https://pypi.org/project/stsauth/). The readme explains how to install/upgrade. Is that what you're asking for?

**Comment by Musikolo** [comment]
Well, I meant releases of the source code [directly from Github](https://github.com/cshamrick/stsauth/releases) same as you can see in many other projects such as [this one](https://github.com/SoftInstigate/restheart/releases) or [this one](https://github.com/mongodb/mongo/releases) among many others.

I'm distributing your code in form of an [Arch Linux package](https://aur.archlinux.org/packages/stsauth/), and that would help me keep a better release control.

Thank you!

**Comment by cshamrick** [comment]
@Musikolo understood. Yeah that shouldn't be a problem. I'll release `0.2.0` first chance I get. I'm glad you're interested in the project. Is it working well for your use-case?

**Comment by Musikolo** [comment]
Yes, it's working like a charm. Very nice tool. You are doing a great job!

Thank you!

**Comment by Musikolo** [comment]
Excellent!

Thank you so much!
